keras_cv/models/backbones/efficientnetv2/efficientnet_v2_backbone.py

```diff
@@ -831,246 +739,257 @@ def __init__(
-            "pooling": self.pooling,
-            "num_classes": self.num_classes,
-            "classifier_activation": self.classifier_activation,
-            "trainable": self.trainable,
-        }
-
-    @classmethod
-    def from_config(cls, config):
-        return cls(**config)
-
-
-def EfficientNetV2B0(
-    *,
-    include_rescaling,
-    include_top,
-    weights=None,
-    input_shape=(None, None, 3),
-    input_tensor=None,
-    pooling=None,
-    num_classes=None,
-    classifier_activation="softmax",
-    **kwargs,
-):
-    return EfficientNetV2(
-        include_rescaling=include_rescaling,
-        include_top=include_top,
-        width_coefficient=1.0,
-        depth_coefficient=1.0,
-        default_size=224,
-        model_name="efficientnetv2-b0",
-        weights=parse_weights(weights, include_top, "efficientnetv2b0"),
-        input_shape=input_shape,
-        input_tensor=input_tensor,
-        pooling=pooling,
-        num_classes=num_classes,
-        classifier_activation=classifier_activation,
-        **kwargs,
-    )
+        config = super().get_config()
+        config.update(
+            {
+                "include_rescaling": self.include_rescaling,
+                "width_coefficient": self.width_coefficient,
+                "depth_coefficient": self.depth_coefficient,
+                "default_size": self.default_size,
+                "dropout_rate": self.dropout_rate,
+                "drop_connect_rate": self.drop_connect_rate,
+                "depth_divisor": self.depth_divisor,
+                "min_depth": self.min_depth,
+                "bn_momentum": self.bn_momentum,
+                "activation": self.activation,
+                "blocks_args": self.blocks_args,
+                "model_name": self.model_name,
+                "input_shape": self.input_shape[1:],
+                "input_tensor": self.input_tensor,
+            }
+        )
+        return config
 
+    @classproperty
+    def presets(cls):
+        """Dictionary of preset names and configurations."""
+        return copy.deepcopy(backbone_presets)
 
-def EfficientNetV2B1(
-    *,
-    include_rescaling,
-    include_top,
-    weights=None,
-    input_shape=(None, None, 3),
-    input_tensor=None,
-    pooling=None,
-    num_classes=None,
-    classifier_activation="softmax",
-    **kwargs,
-):
-    return EfficientNetV2(
-        include_rescaling=include_rescaling,
-        include_top=include_top,
-        width_coefficient=1.0,
-        depth_coefficient=1.1,
-        default_size=240,
-        model_name="efficientnetv2-b1",
-        weights=parse_weights(weights, include_top, "efficientnetv2b1"),
-        input_shape=input_shape,
-        input_tensor=input_tensor,
-        pooling=pooling,
-        num_classes=num_classes,
-        classifier_activation=classifier_activation,
+
+class EfficientNetV2SBackbone(EfficientNetV2Backbone):
+    def __new__(
+        cls,
+        include_rescaling=True,
```

Comment:
these subclasses should be passing the __init__ constructor of `super()` values for width_coefficient and and depth_coefficient.

See line 912 in the current file:

```

    return EfficientNetV2(
        include_rescaling=include_rescaling,
        include_top=include_top,
        width_coefficient=1.0,
        depth_coefficient=1.1,
        default_size=240,
        model_name="efficientnetv2-b1",
        weights=parse_weights(weights, include_top, "efficientnetv2b1"),
        input_shape=input_shape,
        input_tensor=input_tensor,
        pooling=pooling,
        num_classes=num_classes,
        classifier_activation=classifier_activation,
```

You'll want to preserve this!